backend/test/instance.js

```diff
@@ -101,10 +98,12 @@ describe("GET /api/instance/id", function () {
         .then((res) => {
           let found = false;
           expect(res).to.have.status(200);
-          if (res.body.name === instanceId) {
-            found = true;
-            expect(res.body.name).to.be.equals("test single get");
-            expect(res.body.type).to.be.equals("test single type get");
+          for (let instance of res.body) {
```

Comment:
This route shouldn't be returning an array. I found out earlier that the instance id wasn't being passed properly and it was defaulting to returning the entire. In other words, currently, the `api/instance` route has the same functionality as `api/instance/id`. Update this so it open returns the instance with the requested ID